src/cmd.ts

```diff
@@ -123,6 +123,17 @@ function createCommandWithSharedOptions(name: string, description: string) {
 				.choices(Object.values(MatchMode))
 				.makeOptionMandatory(),
 		)
+		.option(
+			"--no-skip-recheck",
+			"Recheck every torrent before resuming. Leave this off to only recheck those that are necessary.",
+			fallback(fileConfig.skipRecheck, true),
+		)
+		.option(
+			"--max-remaining-for-resume <decimal>",
```

Comment:
This name also seems a bit clunky, although I have been thinking about it for a bit and haven't come up with anything super satisfying. Playing with ideas like `maxDlToIncur` `maxDownload`  `maxToDownload` `autoResumeMaxDownload` feels like the word "download" might help the user understand what it means. Or maybe something like `autoResumeUnder`

Reply:
I think `autoResume` is better than `maxRemaining` as it's more specific, the latter could be misinterpreted to affect getting matches. I think `autoResumeMaxDownload` is best.